src/dataExplorer/components/Sidebar.scss

```diff
@@ -16,6 +33,68 @@
   left: 0;
   right: 0;
 
+  .flux-library-original {
+    height: calc(100% - 30px); // remove the ugly white scroll bar
```

Comment:
I believe you are talking about this scroll bar?

https://user-images.githubusercontent.com/10232835/194182331-9560cf57-6caf-4bce-a911-625eebc301a9.mov



Assuming yes...then both of the two instances of `height: calc(100% - 30px); // remove the ugly white scroll bar`, refer to the same scroll bar on the outer container. That scroll bar can be removed by hiding the overflow on that outer container. Setting `overflow-y: hidden` on the `container-right-side-bar`.

Reply:
Thanks! Setting the `overflow-y: hidden` on the `container-right-side-bar` does makes the code straightforward. Since there is already `overflow-x: hidden` in `container-right-side-bar`, I just combined them to be `overflow: hidden`. 

I still kept `height: 100%` without the `calc()` in the two instances to show the flux function list

Reply:
Actually, we still need the `calc()` to be there for the two instances. Otherwise, the items at the bottom of the flux library will be cut off. See video below

https://user-images.githubusercontent.com/14298407/194342694-6795511d-61c2-4ef2-b8a6-8804ce41a165.mov

Reply:
blargh. ok then. Thanks for testing, and checking the details.